src/pages/learn/react/accessibility/index.mdx

```diff
@@ -1,6 +1,11 @@
----
-layout: 'ultimate-guide'
-title: 'React Accessibility Ultimate Guide'
----
+export const meta = {
```

Comment:
for me, this approach to frontmatter is fine and actually preferred because it doesn't require an oddball mix of YAML syntax and we just stick to typescript FTW.

Reply:
Yah, yaml is 💩